libs/core/src/ports/port.ts

```diff
@@ -63,10 +68,13 @@ const disposeAndExit = async (code: ExitCode = 'UNIT_TEST'): Promise<void> => {
   );
   adapters.clear();
 
-  // exit when not unit testing
-  config.NODE_ENV !== 'test' &&
-    code !== 'UNIT_TEST' &&
-    process.exit(code === 'ERROR' ? 1 : 0);
+  if (config.NODE_ENV !== 'test' && code !== 'UNIT_TEST') {
+    rollbar.wait(() => {
+      log.info('Rollbar logs flushed');
+      // eslint-disable-next-line n/no-process-exit
+      process.exit(code === 'ERROR' ? 1 : 0);
+    });
+  }
```

Comment:
Does `wait` still work properly if the rollbar env is not set (locally)?

Reply:
Good question - lemme try that

Reply:
Tested this and as assumed, `wait` still works properly if `rollbar` is not configured.